src/dndbeyond/content-scripts/character.js

```diff
@@ -183,6 +190,15 @@ async function rollSkillCheck(paneClass) {
         roll_properties.modifier += character._proficiency;
     }
 
+    // effects
+    if(roll_properties.ability == "STR" && character.getSetting("effects-enlarge", false)) {
+        adjustRollAndKeyModifiersWithAdvantage(roll_properties);
+        addEffect(roll_properties, "Enlarge");
+    } else if(roll_properties.ability == "STR" && character.getSetting("effects-reduce", false)) {
+        adjustRollAndKeyModifiersWithDisadvantage(roll_properties);
+        addEffect(roll_properties, "Reduce");
+    }
```

Comment:
Forgot to remove the enlarge/reduce effect on the skill checks now that you moved it to `sendRoll`